ts/types/EntityStats.d.ts

```diff
@@ -3,6 +3,8 @@ declare interface EntityStats {
 	name: string;
 	currentBody: {
 		jointType: string;
+		width: number;
```

Comment:
didn't we decide to use bounds2d?

Reply:
we use bounds2d now, yes, you think we should remove width from types?

Reply:
if we put in work to figure out types of data and objects in taro, I would keep it in as it might be useful later, no need to dismiss the effort even if it is not used currently